src/ui/components/footer/footer.tsx

```diff
@@ -31,10 +36,12 @@ export const LOGO_ICON_LIGHT =
 export const LOGO_ICON_DARK =
   'https://raw.githubusercontent.com/supertokens/supertokens-logo/master/logo/ST_icon_dark_theme.svg'
 
-export const Footer = ({ horizontalAlignment, verticalAlignment, colorMode, size }: FooterProps) => {
```

Comment:
<img width="1438" alt="Screenshot 2022-08-19 at 4 20 22 PM" src="https://user-images.githubusercontent.com/18233774/185603323-38f4601d-3fb7-46f0-ae15-20ad47052eb7.png">

The footer should have some spacing at the top, the top spacing should use the same background as screen background

Reply:
Added the 12px spacing at top.

Spacing only works in sticky mode. 
Page that has image bg like auth page, is not affected by the change.